aliases/git/nu_alias_git.nu

```diff
@@ -0,0 +1,202 @@
+#
+# Aliases
+# (sorted alphabetically)
+#
+
+export alias g                     = git
+
+export alias ga                    = git add
+export alias gaa                   = git add --all
+export alias gapa                  = git add --patch
+export alias gau                   = git add --update
+export alias gav                   = git add --verbose
+export alias gap                   = git apply
+export alias gapt                  = git apply --3way
+
+export alias gb                    = git branch
+export alias gba                   = git branch -a
+export alias gbd                   = git branch -d
+export alias gbda                  = (git branch --no-color --merged | grep -vE "^([+*]|\s*($(git_main_branch)|$(git_develop_branch))\s*$)" | do -i {xargs git branch -d})
```

Comment:
It would be good to have these aliases without externals like grep and xargs. Some nushell users use Windows too.

Reply:
good point! Hm... or should I add some additional checks which adds those aliases if and only if those "extra-binaries" exists?

Reply:
I'm not necessarily against using other externals, but I would prefer for these to be cross-platform since that's one major selling point of nushell. However, if you can easily do things in nushell, where you don't require externals, that would be best.

Reply:
ok, I'll look what I can do!